### ISSUE TITLE
Feature request: Add Terminal Shortcuts to dedicated Tabs

### ISSUE THREAD

**Issue by Enricone27** [sonnyp/Tangram#193]
My Idea is that you have Terminal Command to a specific Tab. So that you can open instantly your Webapp without navigating to it.

**Comment by sonnyp** [comment]
You mean something like

```sh
re.sonny.Tangram --instance-id="123"
```

?

**Comment by Enricone27** [comment]
Yeah, and each Tab has an ID. For example Instagram is on Tab nr 5. If you want to lunch Instagram you can start Tangram and Instagram is alredy there so that you don't need to first switch to the Instagram Tab

**Comment by imhemish** [comment]
It has great usecase: one can create desktop files for their pinned apps and can create link handler; so when i open a link in some other app, it would open in respective tabs.

**Comment by sonnyp** [comment]
Epiphany can already create a desktop file for a webapp https://fedoramagazine.org/standalone-web-applications-gnome-web/

**Comment by imhemish** [comment]
> Epiphany can already create a desktop file for a webapp https://fedoramagazine.org/standalone-web-applications-gnome-web/

Not any more, they still have the code inside, but the options and button to create web apps is not there and i found out they have removed it till they get better support, plus they would also integrate PWAs (someone is already working on it as seen in a youtube video on LinuxAppSummit channel)

**Comment by sonnyp** [comment]
@imhemish 

This is in the menu of Epiphany / Web 43.0

![image](https://user-images.githubusercontent.com/19673/210882777-45e2e58a-02e7-4fe5-9c5f-c871b78ad8c9.png)

**Comment by imhemish** [comment]
> @imhemish
> 
> This is in the menu of Epiphany / Web 43.0
> 
> ![image](https://user-images.githubusercontent.com/19673/210882777-45e2e58a-02e7-4fe5-9c5f-c871b78ad8c9.png)

Strange, not visible on my system, running web 43

**Comment by sonnyp** [comment]
Consider installing Epiphany from Flathub
